zerver/lib/actions.py

```diff
@@ -4801,8 +4803,14 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
     if (len(changed_messages) > 0 and new_stream is not None and
             stream_being_edited is not None):
         # Notify users that the topic was moved.
-        notify_topic_moved_streams(user_profile, stream_being_edited, orig_topic_name,
-                                   new_stream, topic_name, send_notification_to_old_thread,
+        old_thread_notification_string = _("This topic was moved by {user} to {new_location}")
+        new_thread_notification_string = _("This topic was moved here from {old_location} by {user}")
+        notify_topic_moved_streams(user_profile,
+                                   stream_being_edited, orig_topic_name,
```

Comment:
A minor formatting things--we are eventually moving toward just having a single argument per line for function calls using 3+ arguments.  It takes a little more vertical space, but it's more diff-friendly, and it makes it easier for most people to visually scan the arguments.

Reply:
Sure, I'll do that!